plugins/adr/src/components/EntityAdrContent/EntityAdrContent.tsx

```diff
@@ -170,6 +175,10 @@ export const EntityAdrContent = (props: {
   const adrApi = useApi(adrApiRef);
   const entityHasAdrs = isAdrAvailable(entity);
 
+  const apiHolder = useApiHolder();
+  const config = apiHolder.get(configApiRef);
```

Comment:
You can just use the `useApi` hook here to grab the config api instead of directly using `useApiHolder`
```suggestion
  const config = useApi(configApiRef);
```